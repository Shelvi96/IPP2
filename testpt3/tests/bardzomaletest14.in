NEW ID

444>340
:0034563>818:572369
570938760>13;167:5;2
0239>9577902119
932885>137
8>62309
2759262>735
40>58851;05
;32>4;6::4702
02>9;19269:
;94493>3943721
16>3:8
8715183>7203:26:
728742::>81241:328
237>9
@72467;08:9217690861;177:
@290;95:39831933:545365;
@:
@551733572615221532870
@39896;14272365;;51:6:400
@829376:468;513;8973366
@;86:06:
@64
@9;804287092;:06
@703704
@:430::40;89;
@3;;3117
