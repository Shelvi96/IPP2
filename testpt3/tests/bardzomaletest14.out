100000001000
26367915376
0
524288
763683617952
2416410548
0
0
81
0
0
0
